README.md

```diff
@@ -1,7 +1,7 @@
 <img src="https://raw.githubusercontent.com/equinor/pysand/master/resources/logo.png" align="center" title="pySand"/>
 
 [![PyPI](https://img.shields.io/pypi/v/pysand)](https://pypi.org/project/pysand/)
-[![Build status](https://travis-ci.org/equinor/pysand.svg?master)](https://travis-ci.org/equinor/pysand)
+[![Build status](https://travis-ci.org/equinor/pysand.svg?branch=master)](https://travis-ci.org/equinor/pysand)

```

Comment:
What is the rationale? I think the result is the same?

Reply:
I noticed when this branch was failing build tests, that the sticker on master-branch readme.md also showed build failing. When changing to branch=master, the logo showed passing.